### PR TITLE
Added Anitube.in presence

### LINKED ISSUE #3719: Anitube.in | anitube.in
**Prerequisites and essential questions** 
- [x] Is it a popular site?
- [x] Is the website older than 2 months? 
- [ ] Is the site locked to a specific country/region?
- [ ] Is the site a paid service? (e.g. Netflix, Hulu)
- [ ] Does the website feature NSFW content? (e.g. porn, etc...)
- [ ] Are you a donator/patron?
- [x] Do you acknowledge that coding presences is completely voluntary and may take time for your service to be added regardless of priority?

**What's your Discord username?** 
Ericlys Syllas (Pain Tendo)#9748

**What's the name of the service?** 
https://www.anitube.in/

**What should the Presence display?** 
Show that I am on the homepage, what I am watching at the moment, in addition to some secondary pages such as list of anime, dramas, tokusatsus and so on. 

**If possible, please provide a logo for the service (512x512 minimum)** 
- https://i.imgur.com/pL1fgAo.jpg
- https://i.imgur.com/hjxgxyG.png

### PR DESCRIPTION
**Pull request by vahya**
Added presence for Anitube.in, resolves #3719 
Screenshot 2021-06-13 100316

### CODE REVIEW

File: websites/A/AniTubeDotIn/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  },
+  "contributors": [
+    {
+      "name": "Ericlys Syllas (Pain Tendo)",
+      "id": "135434213846548480"
+    }
+  ],
+  "category": "anime",
+  "logo": "https://i.imgur.com/pL1fgAo.jpeg",
+  "thumbnail": "https://i.imgur.com/9N9xI53.jpeg",
```
Comment (Bas950) [inline websites/A/AniTubeDotIn/dist/metadata.json:16]: please take a normal size banner image.

File: websites/A/AniTubeDotIn/dist/metadata.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  "version": "1.0.0",
+  "color": "#0fa6a9",
+  "tags": ["anime", "stream"],
+  "url": "www.anitube.in",
+  "description": {
+    "en": "Anitube is an anime website to stream anime"
```
Comment (Bas950) [inline websites/A/AniTubeDotIn/dist/metadata.json:22]: try getting a better description

File: websites/A/AniTubeDotIn/dist/metadata.json [hunk 3]
```diff
@@ -0,0 +1,24 @@
+{
+  "$schema": "https://schemas.premid.app/metadata/1.3",
+  "service": "AniTubeDotIn",
```
Comment (Bas950) [inline websites/A/AniTubeDotIn/dist/metadata.json:3]: you can use "."

Comment (Bas950) [reply websites/A/AniTubeDotIn/dist/metadata.json:3]: *I think*

**Review by Bas950 — CHANGES_REQUESTED** [review]

**Comment by vahya** [comment]
I've fixed the banner and the service name issue, not sure if the description is fixed but it seems better than before

File: websites/A/AniTube.in/presence.ts [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+
+      presenceData.details = `Watching ${title.substr(9, title.substr(9).indexOf("ep"))}`;
+      presenceData.state = `Episode ${parseInt(title.substr(title.indexOf("ep") + 2))}`;
+      presenceData.buttons = [{
+        label: "Watch Along",
+        url: document.location.href
+      }];
+      if (!(paused === null || paused || isNaN(duration))) {
+        presenceData.state = "Time left";
+        [presenceData.startTimestamp, presenceData.endTimestamp] =
```
Comment (ririxidev) [inline websites/A/AniTube.in/presence.ts:87]:
```suggestion
        [, presenceData.endTimestamp] =
```

File: websites/A/AniTube.in/presence.ts [hunk 5]
```diff
@@ -0,0 +1,98 @@
...
+        duration: number = videoElement !== null ? videoElement.duration : NaN;
+
+      presenceData.details = `Watching ${title.substr(9, title.substr(9).indexOf("ep"))}`;
+      presenceData.state = `Episode ${parseInt(title.substr(title.indexOf("ep") + 2))}`;
+      presenceData.buttons = [{
+        label: "Watch Along",
+        url: document.location.href
+      }];
+      if (!(paused === null || paused || isNaN(duration))) {
+        presenceData.state = "Time left";
```
Comment (ririxidev) [inline websites/A/AniTube.in/presence.ts:86]: for what purpose????????

File: websites/A/AniTube.in/presence.ts [hunk 6]
```diff
@@ -0,0 +1,98 @@
...
+      pathname.startsWith("/tokusatsu/letra/") ||
+      pathname.startsWith("/donghua/letra/")
+    ) {
+      const query: string = pathname.substring(pathname.length - pathname.split("").reverse().join("").indexOf("/"));
+      if (query !== "todos") {
+        presenceData.details = "Looking for Animes";
+        presenceData.state = `starting with ${query}`;
+      } else {
+        presenceData.details = "Looking at all Animes";
+        presenceData.state = "";
```
Comment (ririxidev) [inline websites/A/AniTube.in/presence.ts:55]:
```suggestion
```

File: websites/A/AniTube.in/presence.ts [hunk 7]
```diff
@@ -0,0 +1,98 @@
...
+      presenceData.state = searchQuery.substring(17);
+    } else if (
+      pathname === "/anime" ||
+      pathname === "/animes-dublado" ||
+      pathname === "/doramas" ||
+      pathname === "/tokusatsu" ||
+      pathname === "/donghua"
+    ) {
+      presenceData.details = "Looking at all Animes";
+      presenceData.state = "";
```
Comment (ririxidev) [inline websites/A/AniTube.in/presence.ts:41]:
```suggestion
```

File: websites/A/AniTube.in/presence.ts [hunk 8]
```diff
@@ -0,0 +1,98 @@
...
+    searchQuery: string = search !== null ? search.value : null,
+    { pathname } = document.location;
+
+  if (searchContainer.style.display === "block") {
+    presenceData.details = `Searching for ${searchQuery}`;
+    presenceData.state = "";
+  } else {
+    if (pathname === "/") {
+      presenceData.details = "Browsing Home Page";
+      presenceData.state = "";
```
Comment (ririxidev) [inline websites/A/AniTube.in/presence.ts:25]:
```suggestion
```

**Review by rxri — CHANGES_REQUESTED** [review]
and after doing these changes, fix eslint issues

File: websites/A/AniTube.in/presence.ts (hunk 4 above)
Comment (vahya) [reply websites/A/AniTube.in/presence.ts]: [Docs](https://docs.premid.app/en/dev/presence/class#gettimestampsfrommediahtmlmediaelement) says the first return value can be used as startTimestamp, can I ask when we are supposed to used that and when not?

File: websites/A/AniTube.in/presence.ts (hunk 5 above)
Comment (vahya) [reply websites/A/AniTube.in/presence.ts]: Sorry, this was before I was aware of `getTimestampsfromMedia`, removed.

File: websites/A/AniTube.in/dist/metadata.json [hunk 9]
```diff
@@ -0,0 +1,24 @@
...
+  "author": {
+    "name": "वाह्य",
+    "id": "819696394180100106"
+  },
+  "contributors": [
+    {
+      "name": "Ericlys Syllas (Pain Tendo)",
+      "id": "135434213846548480"
+    }
+  ],
```
Comment (ririxidev) [inline websites/A/AniTube.in/dist/metadata.json:13]:
```suggestion
```

File: websites/A/AniTube.in/dist/metadata.json [hunk 10]
```diff
@@ -0,0 +1,24 @@
+{
+  "$schema": "https://schemas.premid.app/metadata/1.3",
```
Comment (ririxidev) [inline websites/A/AniTube.in/dist/metadata.json:2]:
```suggestion
  "$schema": "https://schemas.premid.app/metadata/1.4",
```

**Review by rxri — CHANGES_REQUESTED** [review]

**Comment by vahya** [comment]
@ririxidev @SlowLife1661 What should I do, the deepscan issues are not in my code?

**Comment by ririxidev** [comment]
Nothing

**Review by Bas950 — DISMISSED** [review]
LGTM

File: websites/A/AniTube.in/dist/metadata.json [hunk 11]
```diff
@@ -0,0 +1,18 @@
+{
+  "$schema": "https://schemas.premid.app/metadata/1.4",
+  "service": "AniTube.in",
+  "author": {
+    "name": "वाह्य",
+    "id": "819696394180100106"
+  },
+  "category": "anime",
+  "logo": "https://i.imgur.com/pL1fgAo.jpeg",
+  "thumbnail": "https://i.imgur.com/nRikMNR.jpeg",
```
Comment (ririxidev) [inline websites/A/AniTube.in/dist/metadata.json:10]: only jpg/png :)

**Review by rxri — CHANGES_REQUESTED** [review]

File: websites/A/AniTube.in/dist/metadata.json (hunk 11 above)
Comment (Bas950) [reply websites/A/AniTube.in/dist/metadata.json:10]: Does that really matter?

Comment (ririxidev) [reply websites/A/AniTube.in/dist/metadata.json:10]: yes, low quality xd

**Review by rxri — APPROVED** [review]

**Review by Slowlife01 — APPROVED** [review]

**Pull request merged**